ada_2_fhir-r4/hg/0.2.0/sturen_verwijzing_of_update_huisarts_naar_paramedicus/fhir_instance/composition-hg-VerwijzingHuisartsParamedicusMinimaalbericht.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Composition xmlns="http://hl7.org/fhir">
+   <id value="composition-hg-VerwijzingHuisartsParamedicusMinimaalbericht"/>
+   <meta>
+      <profile value="http://nictiz.nl/fhir/StructureDefinition/hg-ReferralComposition"/>
+   </meta>
+   <identifier>
+      <value value="hg-VerwijzingHuisartsParamedicusMinimaalbericht"/>
+   </identifier>
+   <status value="final"/>
+   <type>
+      <coding>
+         <system value="http://loinc.org"/>
+         <code value="57133-1"/>
+         <display value="Referral note"/>
+      </coding>
+   </type>
+   <subject>
+      <reference value="Patient/patient-"/>
+      <type value="Patient"/>
+      <display value="Patient,"/>
+   </subject>
+   <date value="T"/>
+   <author>
+      <reference value="PractitionerRole/practitionerrole-000001111"/>
+      <type value="PractitionerRole"/>
+      <display value="Healthcare professional (role)"/>
+   </author>
+   <title value="Referral note"/>
+   <section>
```

Comment:
The .text values are missing in the sections.

Reply:
Solved!